docs/BigQueryTable-batchsource.md

```diff
@@ -45,6 +45,9 @@ If no value is given, all partitions up to the partition end date will be read.
 **Partition End Date**: Exclusive partition end date, specified as 'yyyy-MM-dd'. For example, '2019-01-01'. 
 If no value is given, all partitions up from the partition start date will be read.
 
+**Filter**: The WHERE clause filters out rows by evaluating each row against bool_expression,
```

Comment:
remove underscore from bool_expression and state this as normal English phrase boolean expression